src/mgmt/rpc.h

```diff
@@ -176,6 +176,7 @@ CRT_RPC_DECLARE(mgmt_profile, DAOS_ISEQ_MGMT_PROFILE,
 CRT_RPC_DECLARE(mgmt_tgt_create, DAOS_ISEQ_MGMT_TGT_CREATE,
 		DAOS_OSEQ_MGMT_TGT_CREATE)
 
+/* kccain1: propagate "force" boolean? */
```

Comment:
Seems unnecessary for now, right?

Reply:
Yes. I've removed the "kccain" comments that were just some working notes